openapi/paths/storefront/ready-to-pay.yaml

```
@@ -0,0 +1,32 @@
+post:
+  tags:
+    - Purchase
+  summary: Ready to Pay
+  operationId: StorefrontPostReadyToPay
+  security:
+    - PublishableApiKey: []
+  description: |
+    Get available payment methods for a specific transaction or a purchase.
+
+    The payment methods order shown to a customer **MUST** be the same as the order in the response.
```

```suggestion
    The payment methods order shown to a customer **SHOULD** be the same as the order in the response.
```
I think they won't get in trouble if it isn't (typically)... so I think it's a SHOULD.